app/models/tag.rb

```diff
@@ -23,4 +25,15 @@ class Tag < ApplicationRecord
   enumerize :edit_permission,
             in: %i[root all],
             default: :root
+
+  def self.edit_permission_label(value)
```

Comment:
Please keep view code out of the model and move it to a helper.

Reply:
I disagree that this is view code per se. It's rendering an opaque property of a model (in this case an enum) to a string, for the sake of future editing we should be able to edit and alter this in one single place. In addition, there is precedence for this being within the model as per the Neighbourhoods `shortname`, `contextual_name` values (https://github.com/geeksforsocialchange/PlaceCal/blob/main/app/models/neighbourhood.rb#L34), in addition to other examples of this singleton pattern across the codebase where Enumerize is used.

This belongs here because while it does interact with a value passed from the view, it is rendering a property of a model, that should by itself belong to the model.

Reply:
Solved in standup